src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const listWithImg = document.getElementById('thumbs');
+const mainImg = document.querySelector('.gallery__large-img');
+
+listWithImg.addEventListener('click', clickToImg);
+
+function clickToImg(dataOfEvent) {
+  const linkWithImg = dataOfEvent.target.closest('.list-item__link');
+
+  if (linkWithImg) {
+    mainImg.src = linkWithImg.href;
+  }
+
+  dataOfEvent.preventDefault();
```

Comment:
The `preventDefault()` method should be called only if `linkWithImg` is truthy. Otherwise, it might prevent default actions for other clicks that are not on the links. Consider moving `dataOfEvent.preventDefault();` inside the `if` block.